diff -Nur monopd-0.6.2.orig/conf/Makefile.in monopd-0.6.2/conf/Makefile.in
--- monopd-0.6.2.orig/conf/Makefile.in	Wed Jan  1 02:24:45 2003
+++ monopd-0.6.2/conf/Makefile.in	Wed Jan  1 02:26:24 2003
@@ -70,7 +70,7 @@
 
 EXTRA_DIST = monopd.conf-dist
 
-configdir = $(DESTDIR)$(sysconfdir)
+configdir = $(sysconfdir)
 config_DATA = monopd.conf-dist
 subdir = conf
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
diff -Nur monopd-0.6.2.orig/games/Makefile.in monopd-0.6.2/games/Makefile.in
--- monopd-0.6.2.orig/games/Makefile.in	Wed Jan  1 02:24:45 2003
+++ monopd-0.6.2/games/Makefile.in	Wed Jan  1 02:25:43 2003
@@ -70,7 +70,7 @@
 
 EXTRA_DIST = *.conf
 
-gameconfigdir = $(DESTDIR)$(pkgdatadir)/games
+gameconfigdir = $(pkgdatadir)/games
 gameconfig_DATA = *.conf
 subdir = games
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
